grails-app/domain/org/pih/warehouse/inventory/CycleCountRequest.groovy

```diff
@@ -0,0 +1,61 @@
+package org.pih.warehouse.inventory
+
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.core.User
+import org.pih.warehouse.product.Product
+
+class CycleCountRequest {
+
+    String id
+
+    Location facility
+
+    Product product
+
+    CycleCountRequestStatus status
+
+    // To be included when cycle count table is created
+    // CycleCount cycleCount
```

Comment:
Was it agreed that it can be skipped now? To add it we will have to write a second migration. Would not it be better if we had the core cycle count domain here as well?